processing/src/main/java/io/druid/query/filter/DruidFloatPredicate.java

```diff
@@ -22,9 +22,31 @@
 /**
  * FloatPredicate is only supported in Java 8+, so use this to avoid boxing when a float predicate is needed.
  */
+@SuppressWarnings("unused")
```

Comment:
Same

Reply:
I added this as all implementations of DruidFloatPredicate use lambda and intellij was wrongly complaining that the parameter input is unused.